lib/changeling/models/logling.rb

```diff
@@ -16,9 +16,9 @@ class Logling
       property :modified_at, :type => 'date'
 
       mapping do
-        indexes :klass, :type => "string"
-        indexes :oid, :type => "string"
-        indexes :modified_by, :type => "string"
+        indexes :klass, :type => 'string'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.